web/src/app/services/data-store/data-store.service.ts

```diff
@@ -130,7 +137,11 @@ export class DataStoreService {
   loadAccessibleSurveys$(userEmail: string): Observable<List<Survey>> {
     return this.db
       .collection(SURVEYS_COLLECTION_NAME, ref =>
-        ref.where(new FieldPath('acl', userEmail), 'in', Object.keys(Role))
+        ref.where(
+          new FieldPath(FieldNumbers.Survey.acl, userEmail),
+          'in',
+          [1, 2, 3]
```

Comment:
Also, doesn't excluding the string values from the query immediately break backwards compatibility?